tests/integration_tests/utils/public_interfaces_test.py

```diff
@@ -14,19 +14,16 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-# pylint: disable=no-self-use
+from typing import Any, Callable, Dict
+
 import pytest
 
-from superset.sql_lab import dummy_sql_query_mutator
 from superset.utils.public_interfaces import compute_hash, get_warning_message
-from tests.integration_tests.base_tests import SupersetTestCase
 
 # These are public interfaces exposed by Superset. Make sure
 # to only change the interfaces and update the hashes in new
 # major versions of Superset.
-hashes = {
-    dummy_sql_query_mutator: "Kv%NM3b;7BcpoD2wbPkW",
-}
+hashes: Dict[Callable[..., Any], str] = {}
```

Comment:
There was issues in referencing the `app.config["SQL_QUERY_MUTATOR"] because i) the app was the test rather than the default config, and ii) the test wasn't running within an app context. Furthermore it's unclear why the default SQL query mutator was needed explicit checking compared to say any other of the default interfaces.